### PR TITLE
Ensure all Common includes use a <Link/>

### LINKED ISSUE #144: All the linked files are in the root folder of the XML projects.
When you reference files outside the project directory, such as the common directory in the XML code case, these links by default show up in the root of the project in VS which is very annoying and clutters the solution unnecessarily. We should add the appropriate <Link> metadata to these Compile items so that VS will group them correctly in the solution explorer.

### PR DESCRIPTION
**Pull request by stephentoub**
Modifies various .csproj files to ensure that all Includes for files under Common use a <Link/> so that Visual Studio shows the files under a Common folder rather than flattened in the root.

Fixes #144

### CODE REVIEW

**Comment by stephentoub** [comment]
cc: @weshaggard, @akoeplinger

**Comment by Maxwe11** [comment]
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/System.Net.Http.Tests.csproj#L26

**Comment by stephentoub** [comment]
Thanks, @Maxwe11; I addressed those, too.

**Comment by Maxwe11** [comment]
:+1:

File: src/System.Xml.XmlSerializer/tests/System.Xml.XmlSerializer.Tests.csproj [hunk 1]
```diff
@@ -17,7 +17,9 @@
     <Compile Include="XmlSerializerTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="$(CommonTestPath)\System\Console.cs" />
+    <Compile Include="$(CommonTestPath)\System\Console.cs">
+      <Link>Common\System\Console.cs</Link>
```
Comment (weshaggard) [inline src/System.Xml.XmlSerializer/tests/System.Xml.XmlSerializer.Tests.csproj]: Orthogonal but any ideas whey we need this common Console.cs?

**Comment by weshaggard** [comment]
LGTM thanks for cleaning this up @stephentoub

File: src/System.Xml.XmlSerializer/tests/System.Xml.XmlSerializer.Tests.csproj (hunk 1 above)
Comment (stephentoub) [inline src/System.Xml.XmlSerializer/tests/System.Xml.XmlSerializer.Tests.csproj]: The comment in that file says:
```C#
    // A dummy class for passing through System.Console calls until the System.Console contract is available.
    // To-do: Remove this workaround when possible.
```
and then implements WriteLine to call Debug.WriteLine.

**Comment by stephentoub** [comment]
Thanks for reviewing, Wes.

**Pull request merged**